File: curso-python/ex036.py
# Pergunta o valor da casa
valor_casa = float(input("Qual o valor da casa?: R$").replace("R$", "").replace(".", "").replace(",", ".").strip())

# Pergunta o salário do comprador
salario_comprador = float(input("Qual o seu salário?: R$").replace("R$", "").replace(".", "").replace(",", ".").strip())

# Pergunta em quantos anos será pago
anos_pagamento = int(input("Quantos anos você vai pagar a casa?: "))

# Calcula o valor da prestação
prestacao_mensal = valor_casa / (anos_pagamento * 12)

# Calcula 30% do salário
limite_prestacao = salario_comprador * 0.3

# Verifica se a prestação está dentro do limite
if prestacao_mensal <= limite_prestacao:
    print(f"✅ Empréstimo aprovado! Prestação mensal: R$ {prestacao_mensal:.2f}")
else:
    print(f"❌ Empréstimo negado! Prestação de R$ {prestacao_mensal:.2f} excede 30% do salário (R$ {limite_prestacao:.2f})")

File: javascript-exercicios/ex036.js
const prompt = require("prompt-sync")();

// Pergunta o valor da casa
let valorCasa = prompt("Qual o valor da casa? R$ ").replace("R$", "").replace(".", "").replace(",", ".").trim();
valorCasa = parseFloat(valorCasa);

// Pergunta o salário do comprador
let salarioComprador = prompt("Qual o seu salário? R$ ").replace("R$", "").replace(".", "").replace(",", ".").trim();
salarioComprador = parseFloat(salarioComprador);

// Pergunta em quantos anos será pago
let anosPagamento = parseInt(prompt("Quantos anos você vai pagar a casa? "));

// Calcula o valor da prestação mensal
let prestacaoMensal = valorCasa / (anosPagamento * 12);

// Calcula 30% do salário
let limitePrestacao = salarioComprador * 0.3;

// Verifica se o empréstimo será aprovado
if (prestacaoMensal <= limitePrestacao) {
    console.log(`✅ Empréstimo aprovado! Prestação mensal: R$ ${prestacaoMensal.toFixed(2)}`);
} else {
    console.log(`❌ Empréstimo negado! Prestação de R$ ${prestacaoMensal.toFixed(2)} excede 30% do salário (R$ ${limitePrestacao.toFixed(2)})`);
}
